stylecop.json

```diff
@@ -0,0 +1,24 @@
+﻿{
+  // ACTION REQUIRED: This file was automatically added to your project, but it
+  // will not take effect until additional steps are taken to enable it. See the
+  // following page for additional information:
+  //
+  // https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/EnableConfiguration.md
+
+  "$schema": "https://raw.githubusercontent.com/DotNetAnalyzers/StyleCopAnalyzers/master/StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json",
+  "settings": {
+    "documentationRules": {
+      "companyName": ".NET Foundation",
+      "copyrightText": "Copyright (c) .NET Foundation. All rights reserved.\r\nLicensed under the MIT License. See License.txt in the project root for license information.",
+      "xmlHeader": false,
+      "documentInterfaces": false,
```

Comment:
It does replace the old file, but some of the settings are different and this is a bit reduced.

I marked that as false for now are we have quite a few interfaces that are not documented and I wanted to reduce the amount of actual code modified with this change. I do think it would be beneficial to eventually flip this bit.